Framework/Kernel/src/IPropertyManager.cpp

```diff
@@ -6,6 +6,7 @@
 // SPDX - License - Identifier: GPL - 3.0 +
 #include "MantidKernel/IPropertyManager.h"
 #include "MantidKernel/IPropertySettings.h"
+#include "MantidKernel/Logger.h"
```

Comment:
Could remove the include and logger definition below now that we are not using them?

Reply:
Removed it now